packages/financial-templates-lib/src/logger/PagerDutyV2Transport.ts

```diff
@@ -14,6 +15,8 @@ export class PagerDutyV2Transport extends Transport {
   private readonly integrationKey: string;
   private readonly customServices: { [key: string]: string };
   constructor(winstonOpts: TransportOptions, { integrationKey, customServices = {} }: Config) {
+    // we check this even though its strongly typed because typescript is incorrectly allowing json parsed env.
+    assert(integrationKey, "Pagerduty V2 Transport requires integrationKey");
```

Comment:
it might be better to just throw an error like is done in the catch block of the transport. that way you dont need the extra import and the fail mode is the same as in the try-catch logic.